test/lib/lighthouse.js

```diff
@@ -286,19 +313,39 @@ describe('lighthouse', () => {
         });
       });
 
-      it('outputs the report to the file and the logging output', () => {
+      it('outputs reports to json and xml files and the logging output', () => {
+
         return lighthouseRunner.run().then(() => {
           sandbox.assert.calledOnce(reportBuilder.build);
           sandbox.assert.calledWith(fs.writeFileSync, sandbox.match(/lighthouse-report\.xml$/), 'Built report');
           sandbox.assert.calledWith(colourfulLog.log, 'Built report');
+          sandbox.assert.calledWith(fs.writeFileSync, sandbox.match(ATW_OUTPUT_JSON), sandbox.match('Best Practices'));
```

Comment:
Is there a way we can make this test more specific than checking the output just included "Best Practices"?